apps/web/src/state/index.tsx

```diff
@@ -1,156 +1,51 @@
 
   let contextValue = {
     error,
     setError,
-    isFetching,
     activeSinkId,
     setActiveSinkId,
     settings,
     dispatchSetting,
-    roomType,
     appState,
     appDispatch,
   } as StateContextType;
 
-  if (process.env.REACT_APP_SET_AUTH === 'firebase') {
-    contextValue = {
-      ...contextValue,
-      ...useFirebaseAuth(), // eslint-disable-line react-hooks/rules-of-hooks
-    };
-  } else if (process.env.REACT_APP_SET_AUTH === 'passcode') {
-    contextValue = {
-      ...contextValue,
-      ...usePasscodeAuth(), // eslint-disable-line react-hooks/rules-of-hooks
-    };
-  } else {
-    contextValue = {
-      ...contextValue,
-      getToken: async (user_identity, room_name) => {
-        const endpoint = process.env.REACT_APP_TOKEN_ENDPOINT || '/token';
-
-        return fetch(endpoint, {
-          method: 'POST',
-          headers: {
-            'content-type': 'application/json',
-          },
-          body: JSON.stringify({
-            user_identity,
-            room_name,
-            create_conversation: process.env.REACT_APP_DISABLE_TWILIO_CONVERSATIONS !== 'true',
-          }),
-        }).then(res => res.json());
-      },
-      updateRecordingRules: async (room_sid, rules) => {
-        const endpoint = process.env.REACT_APP_TOKEN_ENDPOINT || '/recordingrules';
-
-        return fetch(endpoint, {
-          headers: {
-            'Content-Type': 'application/json',
-          },
-          body: JSON.stringify({ room_sid, rules }),
-          method: 'POST',
-        })
-          .then(async res => {
-            const jsonResponse = await res.json();
-
-            if (!res.ok) {
-              const recordingError = new Error(
-                jsonResponse.error?.message || 'There was an error updating recording rules'
-              );
-              recordingError.code = jsonResponse.error?.code;
-              return Promise.reject(recordingError);
-            }
-
-            return jsonResponse;
-          })
-          .catch(err => setError(err));
-      },
-    };
-  }
-
-  const getToken: StateContextType['getToken'] = (name, room) => {
-    setIsFetching(true);
-    return contextValue
-      .getToken(name, room)
-      .then(res => {
-        setRoomType(res.room_type);
-        setIsFetching(false);
-        return res;
-      })
-      .catch(err => {
-        setError(err);
-        setIsFetching(false);
-        return Promise.reject(err);
-      });
-  };
-
-  const updateRecordingRules: StateContextType['updateRecordingRules'] = (room_sid, rules) => {
-    setIsFetching(true);
-    return contextValue
-      .updateRecordingRules(room_sid, rules)
-      .then(res => {
-        setIsFetching(false);
-        return res;
-      })
-      .catch(err => {
-        setError(err);
-        setIsFetching(false);
-        return Promise.reject(err);
-      });
+  contextValue = {
+    ...contextValue,
+    ...usePasscodeAuth(),
```

Comment:
Here I think we can move `...usePasscodeAuth()` up to the first `contextValue` declaration. 

```
  const contextValue = {
    error,
    setError,
    activeSinkId,
    setActiveSinkId,
    settings,
    dispatchSetting,
    appState,
    appDispatch,
    ...usePasscodeAuth(),
  } as StateContextType;

  return <StateContext.Provider value={contextValue}>{props.children}</StateContext.Provider>;
  ```
  
  Really nice cleanup in this file!!